way-cooler/config/init.lua

```diff
@@ -25,8 +25,8 @@ config.init_workspaces(workspace_settings) -- Not implemented yet
 -- Background
 --
 --
--- A background can either be a 6 digit hex value or an image path (not yet supported)
-way_cooler.background = 0x5E4055
+-- A background can either be a 6 digit hex value or an image path
+way_cooler.background = "../way-cooler-bg/assets/background.jpg"
```

Comment:
Ha, you got me. I tried to find one in the public domain that looked nice (using [this](https://search.creativecommons.org/)), but couldn't find one I liked. Feel free to find a better one to use, as I'm not sure what this is licensed under. Eventually, we should contract someone (Donbot?) to make one for us.